R/pairwise_pop_fst.R

```diff
@@ -22,6 +22,8 @@
 #'   analysis of population structure. Evolution, 38(6): 1358–1370.
 #'
 #' @param .x a grouped [`gen_tibble`] (as obtained by using [dplyr::group_by()])
+#' @param tidy boolean whether to return a tidy tibble. Default is TRUE, FALSE
+#'   returns a matrix. Set to TRUE if by_locus is TRUE.
```

Comment:
Not really, now that we have an error message. Something like "It has to be TRUE if `by_locus`=TRUE"